tests-invoke

```diff
@@ -55,6 +55,10 @@ def main():
     # TEST_BRANCH would be `rhel-8.0`
     test_branch = os.environ.get("TEST_BRANCH")
 
+    if opts.publish and not revision:
+        sys.stderr.write("TEST_REVISION is required for proper publishing")
```

Comment:
Missing `\n`. However, it's IMHO better to use `parser.error("TEST_REVISION is required for proper publishing")` (without `\n`), that also exits the program.

Reply:
`parser.error()` is cool and useful! Shame I didn't know about it. Used that :)